samples/snippets/README.rst

```diff
@@ -0,0 +1,191 @@
+
+.. This file is automatically generated. Do not edit this file directly.
+
+Stackdriver Logging Python Samples
```

Comment:
this is outdated branding. We should call it Cloud Logging now

Reply:
+1

Reply:
Fixed